Prace_domowe/Praca domowa2/Grupa1/Sebastian_Deregowski/PD2_Sebastian_Deregowski.ipynb

```diff
@@ -0,0 +1,1544 @@
+{
```

Comment:
<p>To prawda :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>